content/authors/alexmcsw/_index.md

```diff
@@ -0,0 +1,44 @@
+---
+# Display name
+title: Alexandra McSween
+
+# Username (this should match your github username)
+authors:
+- alexmcsw
+
+superuser: false
+
+# Role/position e.g. Graduate student
+role: MSc Student
+
+# Organization membership
+Organizations/Affiliations:
+organizations:
+- name: University of Ottawa
+  url: https://www.uottawa.ca/en
+
+# A list of your interests, add as many as you want
+interests:
+- Category Theory
+- Representation Theory
+- Women's History of Math
+
+# Your educational history, add as many as you want
+education:
+  courses:
+  - course: Honours Bachelor of Science in Mathematics
+    institution: University of Ottawa
+    - course: Master's of Science in Mathematics
```

Comment:
Hi Alex,

This line appears to be the issue. Can you try removing the space before `- course` to have both at the same indentation level? 

Is the syntax indent-sensitive @ianabc ?

Reply:
It looks like indentation **is** important in YAML.

Reply:
Fixed, thanks!